tests/Test/Compiler.hs

```diff
@@ -53,21 +52,21 @@ testIf handleResult filePaths =
   where
     setupTest filePath =
       do  source <- readFile filePath
-          let result = Compiler.compile "elm-lang" "core" source Map.empty
+          let result = Compiler.compile "elm-lang" "core" True source Map.empty
           return (testCase filePath (handleResult result))
 
 
 -- CHECK RESULTS
 
-isSuccess :: Either String a -> Assertion
-isSuccess result =
+isSuccess :: ([Compiler.Warning], Either [Compiler.Error] a) -> Assertion
+isSuccess (_, result) =
     case result of
-      Right _ -> assertBool "" True
-      Left msg -> assertFailure msg
+      Right _     -> assertBool "" True
+      Left errors -> assertFailure $ (unlines $ map (Compiler.errorToString "" "") errors)
```

Comment:
Is there a better way to convert the errors to strings? I'm not sure what to pass to `Compiler.errorToString`.